stdlib/2/dircache.pyi

```diff
@@ -0,0 +1,10 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/dircache.py
+
+from typing import List, Text
+
+def reset() -> None: ...
+def listdir(path: Text) -> List[Text]: ...
```

Comment:
I think this actually returns a `List[str]` (regardless of whether `path` is str or unicode).